src/EmojiInput.js

```diff
@@ -298,7 +301,7 @@ class EmojiInput extends React.PureComponent {
             .map((v, k) => [
                 { char: category[k].key, categoryMarker: true, ...category[k] }
             ]);
-        _(emoji)
```

Comment:
nit: rename it to emojis

Reply:
I have changed the param of function to `emojis` instead of `emoji` - let me know if this is what You meant.